dashboard/next.config.js

```diff
@@ -0,0 +1,4 @@
+/** @type {import('next').NextConfig} */
```

Comment:
What does this do?

Reply:
strict mode is like a debug mode, where you get more informative logging about potential issues in the code.

Reply:
alright